go/libraries/doltcore/doltdb/errors.go

```diff
@@ -17,6 +17,8 @@ package doltdb
 import (
 	"errors"
 	"fmt"
+
+	errors2 "gopkg.in/src-d/go-errors.v1"
```

Comment:
Is there any existing naming pattern we've used when we need to import both `errors` and `"gopkg.in/src-d/go-errors.v1`? 

`errors2` just looked a little funny with the "errors.v1" part of the import. 

I see `goerrors` used for the newer go-errors.v1 module used in a couple places in the codebase and would be more consistent here.